src/actions/referral_actions.ts

```
@@ -1,6 +1,9 @@
 "use server";
 
 import apiClient from "@/apiClient/apiClient";
+import { ICoupon } from "@/helpers/types";
+import { getCookie } from "./cookie_actions";
+
 import { TReferralStats, ICoupon } from "@/helpers/types";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ICoupon'. Consider to delete it or rename it.

'ICoupon' is defined here:


(lint/suspicious/noRedeclare)